akka-cluster-tools/src/main/resources/reference.conf

```diff
@@ -72,6 +72,20 @@ akka.cluster.client.receptionist {
   # If specified you need to define the settings of the actual dispatcher.
   use-dispatcher = ""
 
+  # How often failure detection heartbeat messages should be received for
+  # each ClusterClient
+  heartbeat-interval = 2s
+
+  # Number of potentially lost/delayed heartbeats that will be
+  # accepted before considering it to be an anomaly.
+  # The ClusterReceptionist is using the akka.remote.DeadlineFailureDetector, which
+  # will trigger if there are no heartbeats within the duration
+  # heartbeat-interval + acceptable-heartbeat-pause, i.e. 15 seconds with
+  # the default settings.
+  acceptable-heartbeat-pause = 13s
+
+  # Failure detection checking interval for checking all ClusterClients
+  failure-detection-interval = 2s
```

Comment:
I was thinking that this might require tuning in relation to how many nodes will be around. If you have 10,000 nodes then the polling interval may need to be different… don’t you think?

> On 9 May 2016, at 6:31 PM, Patrik Nordwall <notifications@github.com> wrote:
> 
> In akka-cluster-tools/src/main/resources/reference.conf <https://github.com/akka/akka/pull/20462#discussion_r62464486>:
> 
> > @@ -72,6 +72,20 @@ akka.cluster.client.receptionist {
> >    # If specified you need to define the settings of the actual dispatcher.
> >    use-dispatcher = ""
> >  
> > +  # How often failure detection heartbeat messages should be received for
> > +  # each ClusterClient
> > +  heartbeat-interval = 2s
> > +
> > +  # Number of potentially lost/delayed heartbeats that will be
> > +  # accepted before considering it to be an anomaly.
> > +  # The ClusterReceptionist is using the akka.remote.DeadlineFailureDetector, which
> > +  # will trigger if there are no heartbeats within the duration
> > +  # heartbeat-interval + acceptable-heartbeat-pause, i.e. 15 seconds with
> > +  # the default settings.
> > +  acceptable-heartbeat-pause = 13s
> > +
> > +  # Failure detection checking interval for checking all ClusterClients
> > +  failure-detection-interval = 2s
> but does the user really care? when will you change it?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/akka/akka/pull/20462/files/22b7e59e609b89af8a35659692e7f8612bb66c33#r62464486>